src/ProgramBlockEditor.scss

```diff
@@ -95,7 +98,35 @@ button.btn.ProgramBlockEditor__program-deleteAll-button--pressed.ProgramBlockEdi
     display: block;
 }
 
+.ProgramBlockEditor__character-column {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    grid-column-start: 1;
+    grid-column-end: 2;
+    background: #FFF;
+    border-width: 0.1rem 0 0.1rem 0.1rem;
+    border-color: #8F8F8F;
+    border-style: solid;
+}
+
+.ProgramBlockEditor__character-column-character-container {
+    border: 0.1rem solid #8F8F8F;
+    border-radius: 50%;
+    padding: 0.5rem;
+    background-color: #FFF;
+    box-shadow: 0 0 1rem rgba(0, 0, 0, 0.4);
```

Comment:
Could you make this box shadow a little more subtle? A lighter grey or increased opacity.